### DISCUSSION TITLE
TimeTable

### DISCUSSION THREAD

**Discussion by Mephisto090** [mcuiobroker/docuJarvis#4] [Ideas]
Du gibst schön viel Quellcode für Copy&Paste (was ich für Leute die keine Ahnung haben echt gut finde. Lernt man zwar nichts kommt aber zum Ziel). Oft fehlt aber irgendwie eine Beschreibung für die Leute die sich doch mehr damit beschäftigen wollen.

https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/system-effektprog./timetable

Hier ein Vorschlag für eine Einleitung:

In diesem timeTable Beispiel geht es um das Steuern eines Saug- oder Mähroboter an einzelne Wochentagen. Somit lassen sich einzelne Wochentage "An" und "Aus" schalten. Das Beispiel ist speziell für ein Mähroboter, einige Daten können bei Saugroboter weggelassen werden.

On: Einschaltzeit
Off: Endzeit
XXX min: Dauer in Minuten
Opt: Rasenkanten aktiviert
WK: Hat der Roboter an diesem Tag gearbeitet

Was sagst du dazu? Gerne kannst du noch etwas anpassen.

**Comment by Mephisto090** [comment]
```
On:      Einschaltzeit
Off:     Endzeit
XXX min: Dauer in Minuten
Opt:     Rasenkanten aktivieren
WK:      Hat der Roboter an diesem Tag gearbeitet
```

Das bezieht sich auf das Bild unter "meinem" Text :)

**Reply by mcuiobroker** [reply]
Welches Bild?

**Reply by Mephisto090** [reply]
https://3270698361-files.gitbook.io/~/files/v0/b/gitbook-x-prod.appspot.com/o/spaces%2F-Mgj8AE5HtJg1UpnK1cN%2Fuploads%2FgSw1YGYqaEAO7OYhEKBw%2Fimage.png?alt=media&token=cefb49b0-e191-412d-b965-e1ab49081b05

**Comment by mcuiobroker** [comment]
Doku aktualisiert.
